app/instance-initializers/violations-helper.js

```diff
@@ -0,0 +1,11 @@
+import ViolationsHelper from 'ember-a11y-testing/utils/violations-helper';
+
+export function initialize(application) {
+  window.violationsHelper = new ViolationsHelper();
```

Comment:
I don't believe so. Violations are pushed into the helper during audit:
https://github.com/ksin/ember-a11y-testing/blob/62c81262c68a635f953d2a0a0795f417c9bfdcc9/app/instance-initializers/axe-component.js#L87